napalm_logs/config/huawei/init.yml

```diff
@@ -0,0 +1,19 @@
+# Prefix profiler for Huawei devices.
+# This profiler matches messages having the following form:
+#
+# .. code-block:: text
+#
+#   2018-7-19 11:58:03.150.1 this-is-my-hostname %%01LLDP/4/BAD_PACKET(l)[87319]:4 invalid packets were received after
+#   latest notification. The last invalid packet came from interface Ethernet0/0/25.
+#
+prefixes:
+  - time_format: "%Y-%m-%d %H:%M:%S"
+    values:
+      date: (\d+-\d+-\d+)
+      time: (\d+:\d+:\d+)
+      milliseconds: (\d{3})
+      host: ([^ ]+)
+      tag: (\w+\/\w+\/\w+)
+      otherThings: (\(\w+\))
```

Comment:
LOL you really let it like that... I'd suggest renaming this to something else please. :D

Reply:
Done :grin: (I used http://support.huawei.com/enterprise/en/doc/EDOC0100533434)

Reply:
Makes sense, thanks!